lib/cypress/go_import.rb

```diff
@@ -1,35 +1,11 @@
 module Cypress
   module GoImport
-    def self.update_conditions(record)
```

Comment:
Do we not need to update conditions anymore? Or are we doing this on the `cdatools` side?